### ISSUE TITLE
Remove particular Bitmap and File cache

### ISSUE THREAD

**Issue by ritesh-dubey** [koush/ion#517]
How can I remove a bitmap or file cache from memory so that the new Image can be fetched from server.

**Comment by koush** [comment]
Use .noCache() on the request.

**Comment by ritesh-dubey** [comment]
Thanks for the reply Koush. But my doubt was to know if I have already cached an image before and wants to fetch a new image with the same URL then what would be an appropriate solution?

I found the below solution on stackoverflow to remove image cache when there was an updated image.

String key = MyApplication.ION.build(MaraApplication.getInstance())
						.load(url).asCachedBitmap().key;
MyApplication.ION.getCache().remove(key);

This solution used to work and remove the cached image and I could fetch the new image but there was an issue when the images were fetched in listview adapter. The images in a listview used to show the old image first and then changes to the latest image after few seconds. I don't know what am I doing wrong.
